SKILL.tsv

```diff
@@ -5440,29 +5440,29 @@ SKILL_20151022_005439	$Increases the damage dealt on an enemy with [Esquive Touc
 SKILL_20151022_005440	$Flanconnade: Enhance
 SKILL_20151022_005441	$Increases the damage dealt on an enemy with [Flanconnade] by 1% per attribute level.
 SKILL_20151022_005442	$Summon Servant: Quick Order
-SKILL_20151022_005443	
+SKILL_20151022_005443	$The servant applies beneficial effects more often.
 SKILL_20151022_005444	$Slow: Nullify Decrease
-SKILL_20151022_005445	
+SKILL_20151022_005445	$Enemies affected by [Slow] get a 6% lower chance per attribute level of nullifying an attack.
 SKILL_20151022_005446	$Haste: Evasion Increase
-SKILL_20151022_005447	
+SKILL_20151022_005447	$When [Haste] is in effect, your evasion rate increases 20 per attribute level.
 SKILL_20151022_005448	$Stop: SP Decrease
-SKILL_20151022_005449	
+SKILL_20151022_005449	$SP consumption of [Stop] decreases by 40.
 SKILL_20151022_005450	
-SKILL_20151022_005451	
+SKILL_20151022_005451	$Player detoxified by [Detoxify] becomes immune to poison for 4 seconds per attribute level.
 SKILL_20151022_005452	$Wugushi: Poison Reserve
 SKILL_20151022_005453	
 SKILL_20151022_005454	$Jincan Gu: Evasion Rate Decrease
-SKILL_20151022_005455	
+SKILL_20151022_005455	$Evasion rate of enemies affected by [Jincan Gu] decreases by 5% per attribute level.
 SKILL_20151022_005456	$Zhendu: Poison Resistance Decrease
-SKILL_20151022_005457	
+SKILL_20151022_005457	$Poison resistance of enemies affected by [Zhendu] decreases by 7 per attribute level.
 SKILL_20151022_005458	$Capturing: Ally Magic Circle
-SKILL_20151022_005459	
+SKILL_20151022_005459	$You can seize not only an enemies' magic circle, but also your allies' with [Capturing].
 SKILL_20151022_005460	$Hovering: Enhance
 SKILL_20151022_005461	$Increases the damage dealt on an enemy with [Hovering] by 1% per attribute level.
 SKILL_20151022_005462	$Pheasant: Enhance
 SKILL_20151022_005463	$Increases the damage dealt on an enemy with [Pheasant] by 1% per attribute level.
 SKILL_20151022_005464	$Circling: Nullify Decrease
-SKILL_20151022_005465	
+SKILL_20151022_005465	$Enemies in the range of [Circling] get a 5% lower chance per attribute level of nullifying the attack.
```

Comment:
Decreases the chance of nullifying the attack by 5% per attribute level when enemies are within the range of [Circling].